src/chart/main/chart.cpp

```diff
@@ -230,13 +230,15 @@ Gen::Marker *Chart::markerAt(const Geom::Point &point) const
 		auto originalPos = coordSys.getOriginal(point);
 
 		for (auto &marker : actPlot->getMarkers()) {
-			auto drawItem = Draw::DrawItem::create(marker,
+			auto drawItem = Draw::DrawItem::createInterpolated(
+				marker,
```

Comment:
space/tab diff. These arguments are continuations of previous command, so it should be use spaces